tests/loginInvalidUser.test.js

```diff
@@ -0,0 +1,20 @@
+const { test, expect } = require('@playwright/test');
+const { LoginPage } = require('../pages/loginPage.js');
+import { generateRandomString } from '../utils/random.js';
+
+const randomUser = 'invalidUser' + generateRandomString(4);
+const randomPassword = generateRandomString(10);
+
+test.describe('Redmine login', () => {
+    test('should not login with random user and password', async ({ page }) => {
```

Comment:
@karenn32 'test' hook must contain the test case title from the documentation and test case's ID.